src/main/java/io/github/techstreet/dfscript/script/action/ScriptActionArgument.java

```diff
@@ -55,10 +53,10 @@ public ScriptActionArgument plural(boolean plural) {
     }
 
     public ScriptActionArgument defaultValue(ScriptValue value) {
-        if(value instanceof ScriptVariable) return this;
+        if (value instanceof ScriptVariable) return this;
```

Comment:
remaining random formatting changes

Reply:
had to mosify that file so the formatting changes remain